$NetBSD: patch-aa,v 1.2 2006/09/14 21:18:44 rillig Exp $

SunPro does not like non-static inline functions. BSD-based operating
systems have <termios.h> instead of <termio.h>.

--- bast.c.orig	2004-10-15 23:58:52.000000000 +0200
+++ bast.c	2006-09-14 23:14:36.540746472 +0200
@@ -1,7 +1,11 @@
 #include <stdlib.h>
 #include <stdio.h>
 #include <ncurses.h>
+#if defined(__linux)
 #include <termio.h>
+#else
+#include <termios.h>
+#endif
 #include <assert.h>
 #include <unistd.h>
 #include <string.h>
@@ -39,7 +43,7 @@ void bast_clear()
 }
 
 
-inline unsigned char *wyx2pointer(char *well, int y, int x)
+static inline unsigned char *wyx2pointer(char *well, int y, int x)
 {
         return well + (y * WELL_WIDTH) + x;
 }
